packages/postgres-database/src/simcore_postgres_database/migration/versions/42ec7816c0b4_computational_collection_runs.py

```diff
@@ -0,0 +1,122 @@
+"""computational collection runs
+
+Revision ID: 42ec7816c0b4
+Revises: d159ac30983c
+Create Date: 2025-07-01 13:30:02.736058+00:00
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "42ec7816c0b4"
+down_revision = "d159ac30983c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "comp_run_collections",
```

Comment:
to go with the other tables, these should be `comp_runs_collections` there are multiple runs inside a collection. also that will show better in the adminer.